cypress/e2e/shared/dashboardsIndex.test.ts

```diff
@@ -153,10 +153,10 @@ describe('Dashboards', () => {
   })
 
   // TODO - fix failing test (fails only in circleci - cloud-e2e-firefox)
-  it.skip('can import as JSON or file', () => {
+  it('can import as JSON or file', () => {
     const checkImportedDashboard = () => {
       // wait for importing done
-      cy.wait(200)
```

Comment:
I'd really rather avoid increasing the wait here to another arbitrary number. Perhaps we could use [cy.intercept](https://docs.cypress.io/api/commands/intercept) or wait on another element that's visible during the import?

Reply:
I had a feeling yesterday, that there was something that could be improved.  I could have maybe reread your best practices guide, before being satisfied that since I'd found an implicit wait in the code, it could just be reused.

Reply:
love it! Thanks for this change!